### PR TITLE
Get RPS and Wallet hosted on Netlify

### PR DESCRIPTION
**Pull request by snario**
~work in progress~

https://app.netlify.com/sites/wallet-statechannels/deploys
https://app.netlify.com/sites/rps-statechannels/deploys

### TODO:
~- [ ] Get a new bot to work~
~- [ ] Get the new bot to work with virtual funding~
~- [ ] Get the new bot put inside the monorepo~
~- [ ] Figure out how to host the new bot... Heroku?~

Will add bots later.

Works with ledger funding now!

### CODE REVIEW

**Comment by snario** [comment]
Reminder that after this is merged to change the branch settings on Netlify so both deploy on `master`. Right now they're configured to this branch.

**Comment by geoknee** [comment]
Screenshot 2019-12-18 at 16 16 28

Could we cut down a little of this noise? I think there is an option for "not displaying rich content" or something like that (see other netlify deploy configs)

**Comment by snario** [comment]
Done!

Screen Shot 2019-12-18 at 11 19 56

File: packages/wallet/config/webpack.config.js [hunk 1]
```diff
@@ -419,16 +419,18 @@ module.exports = function(webpackEnv) {
...
-        resource
-      ) {
-        if (process.env.TARGET_NETWORK !== "development") {
-          resource.request = resource.request.replace(
-            /.*\/build\/contracts/,
-            paths.appPreBuiltContractArtifacts
-          );
-        }
-      }),
+      // TODO: Figure out if we still need this
```
Comment (snario) [inline packages/wallet/config/webpack.config.js]: @lalexgap I don't think we need to do this. On netlify, we have access to the `@statechannels/nitro-protocol` folder and so we can access the files in there. Assuming `nitro-protocol` also publishes those files in the future, we shouldn't ever need it.

**Pull request merged**